gulpfile.js

```diff
@@ -23,6 +25,10 @@ var browserify = require('browserify');
 var reactify = require('reactify');
 var uglify = require('gulp-uglify');
 
+// Flags.
+var isProduction = (argv.p === undefined) ? false : true;
```

Comment:
Yeah I don't know how to do that though. Right now gulp build -p and gulp -p works fine.